xbmc/interfaces/legacy/ListItem.h

```diff
@@ -691,9 +691,8 @@ namespace XBMCAddon
       /// \ingroup python_xbmcgui_listitem
       /// @brief \python_func{ getdescription() }
       ///-----------------------------------------------------------------------
-      /// Returns the description of this PlayListItem.
+      /// Deprecated.
```

Comment:
I know a bit new with the full doxygen usage :)

Maybe add it as `@warning Deprecated`, it bring then a red line before, as example visible here http://alwinesch.github.io/group__python__monitor.html#ga5ee33edfc21fb06cfe99f892da9b234c.